code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -648,6 +648,34 @@
 	M.losebreath += 5
 	return ..()
 
+/datum/reagent/toxin/spewium
+	name = "Spewium"
+	id = "spewium"
+	description = "A powerful emetic, causes uncontrollable vomiting.  May result in vomiting organs at high doses."
+	reagent_state = LIQUID
+	color = "#2f6617" //A sickly green color
+	metabolization_rate = REAGENTS_METABOLISM
+	overdose_threshold = 29
+	toxpwr = 0
+	taste_description = "vomit"
+
+/datum/reagent/toxin/spewium/on_mob_life(mob/living/M)
+	.=..()
+	if(current_cycle >=11 && prob(min(50,current_cycle)) && ishuman(M))
+		var/mob/living/carbon/human/H = M
+		H.vomit(lost_nutrition = 10, blood = prob(10), stun = prob(50), distance = rand(0,4))
+		for(var/datum/reagent/toxin/R in M.reagents.reagent_list)
+			if(R != src)
+				H.reagents.remove_reagent(R.id,1)
```

Comment:
This should probably purge at the same rate calomel does.

Calomel is supposed to be an emetic, so...